bundle/README.md

```diff
@@ -152,3 +152,111 @@ To configure `bookinfo` with a gateway using `Gateway API`:
+## Istio Addons Integrations
+
+Istio can be integrated with other software to provide additional functionality (More information can be found in: https://istio.io/latest/docs/ops/integrations/). The following addons are for demonstration or development purposes only and should not be used in production environments:
+
+### Prometheus
+
+`Prometheus` is an open-source systems monitoring and alerting toolkit. You can use `Prometheus` with the Sail Operator to keep an eye on how healthy Istio and the apps in the service mesh are, for more information, see [Prometheus](https://istio.io/latest/docs/ops/integrations/prometheus/). 
+
+To install Prometheus you will need to follow this steps:
+
+1. Deploy `Prometheus`:
+
+    ```sh
+    $ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/addons/prometheus.yaml
+    ```
+2. Access to `Prometheus`console:
+
+* Option 1: 
+
+    * Expose the `Prometheus` service externally:
+    
+    ```sh
+    $ oc expose service prometheus -n istio-system
+    ```
+    * Get the route of the service and open the URL in the web browser
+    
+    ```sh
+    $ oc get route prometheus -o jsonpath='{.spec.host}' -n istio-system
+    ```
+<!-- TODO: Verify that the istioctl command works -->
+<!-- * Option 2:
+
+    * Use istioctl to access to the `Prometheus` console
+    
+    ```sh
+    $ istioctl dashboard prometheus
+    ``` -->
+
+### Grafana
+
+`Grafana` is an open-source platform for monitoring and observability. You can use `Grafana` with the Sail Operator to configure dahsboards for istio, see [Grafana](https://istio.io/latest/docs/ops/integrations/grafana/) for more information. 
+
+To install Prometheus you will need to follow this steps:
+
+1. Deploy `Grafana`:
+    
+    ```sh
+    $ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/addons/grafana.yaml
+    ```
+
+2. Access to `Grafana`console:
+
+* Option 1: 
+
+    * Expose the `Grafana` service externally
+    
+    ```sh
+    $ oc expose service grafana -n istio-system
+    ```
+    * Get the route of the service and open the URL in the web browser
+    
+    ```sh
+    $ oc get route grafana -o jsonpath='{.spec.host}' -n istio-system
+    ```
+<!-- TODO: Verify that the istioctl command works -->
+<!-- * Option 2:
+    
+    * Use istioctl to access to the `Grafana` console
+    
+    ```sh
+    $ istioctl dashboard grafana
+    ``` -->
+
+### Jaeger
+
+`Jaeger` is an open-source end-to-end distributed tracing system. You can use `Jaeger` with the Sail Operator to monitor and troubleshoot transactions in complex distributed systems, see [Jaeger](https://istio.io/latest/docs/ops/integrations/jaeger/) for more information. 
+
+To install Prometheus you will need to follow this steps:
+
+1. Deploy `Jaeger`:
+    
+    ```sh
+    $ oc apply -f https://raw.githubusercontent.com/istio/istio/master/samples/addons/jaeger.yaml
+    ```
+2. Access to `Jaeger`console using istioctl:
+
+```sh
+$ istioctl dashboard jaeger
```

Comment:
why don't we create a route like we did with prometheus and grafana? This requires installing `istioctl` which this document doesn't mention how.

Reply:
Make sense to me to create the route because istioctl was not working for the other two addons. I'll change this to expose the route and when we figured out why was failing using istioctl for the other two addons I'll add the option for all the addons making the note that it's needed to go and download istioctl